components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/RequestObjectValidatorImpl.java

```diff
@@ -137,6 +145,17 @@ public boolean validateRequestObject(RequestObject requestObject, OAuth2Paramete
         return true;
     }
 
+    private void checkMandatoryParams(RequestObject requestObject) throws RequestObjectException {
```

Comment:
```suggestion
    private void checkFapiMandatedParams(RequestObject requestObject) throws RequestObjectException {
```

Reply:
Also cannot seem to find a unit test for this.

Reply:
Fixed in f409e32a5e8cad00a5dfeaaac027011720f2c534